apps/mobile/src/screens/PassengerDetailScreen.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from 'react';
+import { PassengerDetail } from '@kiwicom/margarita-core';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { type NavigationScreenProp } from 'react-navigation';
+
+type NavigationParams = {||};
+
+type Props = {|
+  +navigation: NavigationScreenProp<NavigationParams>,
+|};
+
+export default function PassengerDetailScreen(props: Props) {
+  return (
+    <View style={styles.container}>
+      <PassengerDetail />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#e5e5e5', // There is no orbit token for this color
```

Comment:
This has been extracted to `@kiwicom/margarita-config`, please import it from there.

Reply:
fixed